lightningd/peer_control.c

```diff
@@ -1308,15 +1308,29 @@ static void activate_peer(struct peer *peer)
 	u8 *msg;
 	struct channel *channel;
 	struct lightningd *ld = peer->ld;
+	/* Avoid thundering herd: after first five, delay by 1 second. */
+	int delay = -5;
```

Comment:
Did you intend to make this variable `static`, or define it in `activate_peers` and pass it in by pointer to this function?

Reply:
Err... yes... I was just checking that review was working? :laughing: 

Thanks, you, um, passed the test!  Fixed...